frontend/src/locales/scripts/en.json5

```diff
@@ -991,7 +991,7 @@
   },
   searchType: {
     image: "Images",
-    audio: "Audio",
+    audio: "Audio track",
```

Comment:
Please revert this change as we are referring to to the collection of all audio tracks so "Audio" is suitable here.